comms/lighthouse/src/server.ts

```diff
@@ -72,6 +91,8 @@ const CURRENT_ETH_NETWORK = process.env.ETH_NETWORK ?? DEFAULT_ETH_NETWORK
 
   const idService = new IdService({ alphabet: idAlphabet, idLength })
 
+  app.options('*', cors(corsOptions))
```

Comment:
```suggestion
  app.use(cors(corsOptions))
```

Cors works at many levels, for all HTTP methods it should inject headers if necessary

Reply:
What it's doing here it's handling the requests with `OPTIONS` method, I configured it for all paths. Anyway, I can change it to only respond on known routes if you consider it better.

Reply:
Cors headers must be present in responses too, not only in the OPTIONS method. Please use `app.use` instead of `app.options`